config/metadata/basic_metadata.yaml

```
@@ -50,6 +50,8 @@ attributes:
     multiple: true
     form:
       primary: false
+    index_keys:
+      - "description_tesim"
```

```suggestion
      - "description_sim"
      - "description_tesim"
```

Should this have the string version as well?  It is pretty common to include both, but the string stores as the exact string typically used for exact matching.  I don't imagine that someone would be looking for an exact match for the entire description.

> String stores a word/sentence as an exact string without performing tokenization etc. Commonly useful for storing exact matches, e.g, for facetting.
> 
> Text typically performs tokenization, and secondary processing (such as lower-casing etc.). Useful for all scenarios when we want to match part of a sentence.

---

From what I can tell description was only every indexed as `_tesim`.  (https://github.com/samvera/hyrax/blob/main/app/indexers/hyrax/basic_metadata_indexer.rb#L7)